specs/005-structured-logging/spec.md

```
@@ -0,0 +1,81 @@
+
+| ファイル | 変更内容 |
+|---------|---------|
+| app/_lib/logger.ts | 新規作成（StructuredLogger クラス） |
+| app/_lib/actions.ts | createBooking にログ統合 |
+| tests/unit/logger.test.ts | ユニットテスト追加 |
+
+## 関連ドキュメント
+
+- [plan.md](plan.md) - 実装計画
+- [tasks.md](tasks.md) - タスク一覧
+- [specs/002-booking-concurrency-control/operations.md](../002-booking-concurrency-control/operations.md) - 運用ガイドライン
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)